packages/contracts-bedrock/test/safe/LivenessModule2.t.sol

```diff
@@ -422,130 +443,6 @@ contract LivenessModule2_Challenge_Test is LivenessModule2_TestInit {
     }
 }
 
-/// @title LivenessModule2_ChangeOwnershipToFallback_Test
-/// @notice Tests the ownership transfer after successful challenge
-contract LivenessModule2_ChangeOwnershipToFallback_Test is LivenessModule2_TestInit {
```

Comment:
This was all moved into the SaferSafes test suite, since it tests functionality that applies to both contracts (ie. needs to clear both the module and safe properly).